### PR TITLE
Initialize vividus-studio extension template

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
```
Comment (valfirst) [inline .vscode/extensions.json]: https://stackoverflow.com/a/4183018/2067574

Comment (uarlouski) [reply .vscode/extensions.json]: https://code.visualstudio.com/docs/languages/json#_json-with-comments

Comment (valfirst) [reply .vscode/extensions.json]: but github doesn't know about VS format, and while reading red lines it's hard to understand, where an actual error is and where a github warning is

Comment (valfirst) [reply .vscode/extensions.json]: approved, because these comments are auto-generated

**Pull request merged**